dataland-backend/src/main/kotlin/org/dataland/datalandbackend/repositories/utils/StoredCompanySearchFilter.kt

```diff
@@ -5,12 +5,12 @@ package org.dataland.datalandbackend.repositories.utils
  * convenient usage of SEPL instructions in the query
  */
 data class StoredCompanySearchFilter(
-    val dataTypeFilter: List<String>,
-    val countryCodeFilter: List<String>,
-    val sectorFilter: List<String>,
-    val searchString: String,
-    val nameOnlyFilter: Boolean,
-    val uploaderIdFilter: List<String>,
+    val dataTypeFilter: List<String> = listOf(),
+    val countryCodeFilter: List<String> = listOf(),
+    val sectorFilter: List<String> = listOf(),
+    val searchString: String = "",
+    val nameOnlyFilter: Boolean = false,
+    val uploaderIdFilter: List<String> = listOf(),
```

Comment:
Meines Erachtens braucht es die default Werte hier nicht =>

Das hier ist die einzige Stelle wo dieses model genutzt wird:

![image](https://github.com/d-fine/Dataland/assets/99255296/2d96d017-e625-47fe-9167-c53694420a83)

Und hier bring `CompanySearchFilter` bereits default Werte mit sich und steckt sie in `searchFilterForJPA`

Oder übersehe ich etwas?

Reply:
Ja, geh ich mit. Die hab ich eingeführt, weil ich die nur teilweise setzen wollte. Aber der Use Case ist nicht mehr vorhanden. Done.

Reply:
Ok nice das passt.

Aber @fsoffel wir hatten am Freitag oder Donnerstag doch auch noch besprochen, dass `StoredCompanySearchFilter.kt` und `CompanySearchFilter.kt` zwei models sind, die sich großteils überschneiden.

Ich glaube wir hatten verinbart, dass wir das zu einem einzelnen model vereinen, das die aktuelle Nutzung beider models zufriedenstellt.

Steht das noch auf der todo Liste?   Erst danach würde ich die conversation hier schließen.

Reply:
Jup. Mach ich.

Reply:
Done

Reply:
👍